lib/sqewer/submitter.rb

```diff
@@ -21,7 +21,13 @@ def submit!(job, **kwargs_for_send)
     else
       serializer.serialize(job)
     end
-    connection.send_message(message_body, **kwargs_for_send)
+    if connection_pool.is_a?(Sqewer::ConnectionMessagebox)
```

Comment:
Can we use a pool always, but with a stand-in `NullPool` when none is configured?

Reply:
This is a bit trickier, because the message box also uses a connection (it acts as a composed object). Given the API around this, this is the best solution I found. If you have something better in mind, I'd be happy to make it work :)